#request
---
#response
int16[] Color 
int16[] U
int16[] V
float32[] X
float32[] Y
float32[] Z
uint8[] object_type 
int16 amount
